### PR TITLE
modified caption related to image, line 6

### PR DESCRIPTION
**Pull request by amd-juharris**
caption update for cute robo-octo-cat

### CODE REVIEW

File: _slides/07-amd-juharris.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Resistance is Cute-ile.
```
Comment (beardofedu) [inline _slides/07-amd-juharris.md:6]: This is great! Nice work! 🤖

File: _slides/07-amd-juharris.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
```
Comment (beardofedu) [inline _slides/07-amd-juharris.md]:
```suggestion
title: "Welcome to amd-juharris' slide!"
```

**Review by beardofedu — CHANGES_REQUESTED** [review]

File: _slides/07-amd-juharris.md [hunk 3]
```diff
@@ -3,7 +3,7 @@ layout: slide
...
 ---
 
-Caption here
+Resistance is Cute-ile.
 
 ![Robotocat](https://octodex.github.com/images/Robotocat.png)
```
Comment (beardofedu) [inline _slides/07-amd-juharris.md:8]: Awesome image

**Review by beardofedu — CHANGES_REQUESTED** [review]
This is an example of a summary comment

File: _slides/07-amd-juharris.md (hunk 1 above)
Comment (amd-juharris) [reply _slides/07-amd-juharris.md:6]: thanks!

**Review by beardofedu — APPROVED** [review]
Awesome work!! 🚢

**Pull request merged**